docs/source/email_setup.rst

```diff
@@ -0,0 +1,115 @@
+Email Notification Setup
+========================
+
+Dallinger can be configured to send email messages when errors occur during a
+running experiment. If this configuration is skipped, messages which
+would otherwise be emailed will be written to the experiment logs instead.
+
+Instructions
+-------------
+Sending email from Dallinger requires 5 configuration settings, described in
+turn below. Like all configuration settings, they can be set up in either
+`.dallingerconfig` in your home directory, or in `config.txt` in the root
+directory of your experiment.
+
+The Config Settings
+~~~~~~~~~~~~~~~~~~~
+
+``smtp_host`` The hostname and port of the SMTP (outgoing email) server through
```

Comment:
I would put the 'smtp_host', 'smtp_username' on its own line, like is done in the configuration text.